recipe/001-filter-source-files.patch

```diff
@@ -0,0 +1,42 @@
+From 95fb180fd56f6dd2985e12ca57e85ef7ade85895 Mon Sep 17 00:00:00 2001
```

Comment:
Why do we need both this patch and source section at test environment?

Reply:
This patch will be included in v0.3.0. What it does it prevents PIP from including C++/Cython source files into the wheel. The source section in the test environment is a completely separate thing: it lists which files from the source directory conda-build needs to copy to the test environment to run the tests.

Reply:
Thank you for the clarification!